saleor/account/migrations/0079_full_channel_access_group_for_openid.py

```diff
@@ -43,6 +47,21 @@ def update_plugin_default_group_name(plugin_conf, group_name):
     plugin_conf.save(update_fields=["configuration"])
 
 
+def create_full_channel_access_group_task(User, Group, group_name):
+    full_channel_access_group, _ = Group.objects.get_or_create(
+        name=group_name, defaults={"restricted_access_to_channels": False}
+    )
+    GroupUser = User.groups.through
+    group_users = GroupUser.objects.filter(group_id=full_channel_access_group.id)
+    users = User.objects.filter(
+        Q(is_staff=True) & ~Exists(group_users.filter(user_id=OuterRef("id")))
+    ).order_by("-pk")
+    user_ids = users.values_list("pk", flat=True)[:BATCH_SIZE]
+    if user_ids:
+        full_channel_access_group.user_set.add(*user_ids)
+        create_full_channel_access_group_task(User, Group, group_name)
```

Comment:
I guess this table is not that big, but I wonder if it will not be better to always do the batching in migration via loop, instead of using recursion, wdy?

Reply:
I'm not sure if it makes a difference, but I can change it to use batching :)

Reply:
Updated